2つのコンテナを内包するpodを作成
```
(⎈ |local:default)takzok% ka chapter05/sample-2pod.yml
pod "sample-2pod" created
(⎈ |local:default)takzok% kg pod
NAME                                 READY     STATUS    RESTARTS   AGE
order1-deployment-86b68b4c5b-25wqv   1/1       Running   0          1d
order1-deployment-86b68b4c5b-98frf   1/1       Running   0          1d
order1-deployment-86b68b4c5b-fnbmg   1/1       Running   0          1d
sample-2pod                          2/2       Running   0          40s
sample-annotations                   1/1       Running   0          1d
sample-label                         1/1       Running   0          1d
sample-pod                           1/1       Running   2          1d
sample-pod1                          1/1       Running   0          1d
(⎈ |local:default)takzok%    
```

nginxを同一Podで動かそうとすると、portがconflictして、2つ目のcontainerが起動しない。
(2つ目のpod名がredis-container-113となっているが、イメージはnignx)
```
(⎈ |local:default)takzok% ka chapter05/sample-2pod-fail.yml
pod "sample-2pod-fail" created
(⎈ |local:default)takzok% kg pod
NAME                                 READY     STATUS    RESTARTS   AGE
order1-deployment-86b68b4c5b-25wqv   1/1       Running   0          1d
order1-deployment-86b68b4c5b-98frf   1/1       Running   0          1d
order1-deployment-86b68b4c5b-fnbmg   1/1       Running   0          1d
sample-2pod                          2/2       Running   0          4m
sample-2pod-fail                     2/2       Running   1          8s
sample-annotations                   1/1       Running   0          1d
sample-label                         1/1       Running   0          1d
sample-pod                           1/1       Running   2          1d
sample-pod1                          1/1       Running   0          1d
(⎈ |local:default)takzok% kg pod sample-2pod-fail
NAME               READY     STATUS             RESTARTS   AGE
sample-2pod-fail   1/2       CrashLoopBackOff   2          43s
(⎈ |local:default)takzok% k logs sample-2pod-fail -c redis-container-113
2019/05/22 00:51:12 [emerg] 1#1: bind() to 0.0.0.0:80 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address already in use)
2019/05/22 00:51:12 [emerg] 1#1: bind() to 0.0.0.0:80 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address already in use)
2019/05/22 00:51:12 [emerg] 1#1: bind() to 0.0.0.0:80 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address already in use)
2019/05/22 00:51:12 [emerg] 1#1: bind() to 0.0.0.0:80 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address already in use)
2019/05/22 00:51:12 [emerg] 1#1: bind() to 0.0.0.0:80 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address already in use)
2019/05/22 00:51:12 [emerg] 1#1: still could not bind()
nginx: [emerg] still could not bind()
(⎈ |local:default)takzok%     
```